nativescript-core/ui/core/view/view.android.ts

```diff
@@ -497,11 +497,16 @@ export class View extends ViewCommon {
     }
 
     @profile
-    public requestLayout(): void {
-        super.requestLayout();
-        if (this.nativeViewProtected) {
-            this.nativeViewProtected.requestLayout();
+    public requestLayout(calledFromChild?:boolean): void {
```

Comment:
Space is needed after the colon in parameter to match the tslint rules
```suggestion
    public requestLayout(calledFromChild?: boolean): void {
```